tsconfig.json

```diff
@@ -16,6 +16,6 @@
     "noEmit": true,
     "jsx": "react-jsx"
   },
-  "include": ["src"],
+  "include": ["src", "env"],
```

Comment:
Is there any purpose to env in tsconfig?

Reply:
no, when i installed npm its automatically configured will remove